manual/Tasks/junitlauncher.html

```diff
@@ -129,6 +129,18 @@ <h3>Parameters</h3>
         <th scope="col">Description</th>
         <th scope="col">Required</th>
     </tr>
+    <tr>
+        <td>includeTags</td>
+        <td>A comma separated list of <a href="https://junit.org/junit5/docs/current/user-guide/#writing-tests-annotations">JUnit 5 tags</a>, describing the tests to include.            
+        </td>
+        <td>No</td>
+    </tr>
+    <tr>
+        <td>excludeTags</td>
+        <td>A comma separated list of <a href="https://junit.org/junit5/docs/current/user-guide/#writing-tests-annotations">JUnit 5 tags</a>, describing the tests to exclude.            
+        </td>
```

Comment:
Can you make a mention that this attribute is available since Ant 1.10.7? See the `outputDir` attribute of the `listener` element for an example of how we do that.